README.md

```diff
@@ -22,36 +22,36 @@ Populate a `.env` file by using the examples from `.env.example`, or follow the
 
 The `.env` file should **never** be committed.
 
-#### Docker
+#### Local
```

Comment:
I've found local to be all I need - is anyone using docker?

Reply:
I do use the docker environment from time to time, just not on my Guardian Macbook as docker is super slow on there